cli/compose/schema/data/config_schema_v3.3.json

```diff
@@ -313,15 +313,15 @@
       "additionalProperties": false,
       "properties": {
         "disable": {"type": "boolean"},
-        "interval": {"type": "string"},
+        "interval": {"type": "string", "format": "duration"},
         "retries": {"type": "number"},
         "test": {
           "oneOf": [
             {"type": "string"},
             {"type": "array", "items": {"type": "string"}}
           ]
         },
-        "timeout": {"type": "string"}
+        "timeout": {"type": "string", "format": "duration"}
```

Comment:
Normally we do not make any changes to previous versions of the schema. They should be frozen. Would you mind reverting this change in the previous versions?

Reply:
Do you think the change on the datatype will impact the parsing for old version?

Reply:
Nope, it wont